client/blocks/importer/wordpress/upgrade-plan/wordpress-cwvtech-report.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@agrullon95, I have one more question for you. =)

Do you remember what was the source of this JSON?
While checking [this report](https://lookerstudio.google.com/u/0/reporting/55bc8fad-44c2-4280-aa0b-5f3f0cd3d2be/page/M6ZPC?params=%7B%22df44%22:%22include%25EE%2580%25800%25EE%2580%2580IN%25EE%2580%2580WordPress%25EE%2580%2580WordPress.com%22%7D), I see some different values for March 2024. Because of the decimals, I suspect it has a more specific source (?).